core/src/Kokkos_View.hpp

```diff
@@ -604,14 +609,16 @@ class View : public ViewTraits<DataType, Properties...> {
   friend class Kokkos::Impl::ViewMapping;
 
  public:
-  typedef ViewTraits<DataType, Properties...> traits;
+  using traits            = ViewTraits<DataType, Properties...>;
+  using view_tracker_type = Kokkos::Impl::ViewTracker<View>;
```

Comment:
Does this need to be public? If so, the name should probably be something like `experimental_view_tracker_type` for now so that end users don't rely on it until we're comfortable making this permanent.